ssl/t1_lib.c

```diff
@@ -502,28 +465,44 @@ static int tls1_check_pkey_comp(SSL *s, EVP_PKEY *pkey)
 }
 
 /* Check a group id matches preferences */
-static int tls1_check_group_id(SSL *s, uint16_t group_id)
+int tls1_check_group_id(SSL *s, uint16_t group_id)
     {
     const uint16_t *groups;
     size_t groups_len;
 
     if (group_id == 0)
         return 0;
 
-    if (!tls_curve_allowed(s, group_id, SSL_SECOP_CURVE_CHECK))
-        return 0;
+    /* If we have a cipher check group matches Suite B preferences */
```

Comment:
Maybe just "check for Suite B compliance" ?  I couldn't quite parse the comment as-is.